src/Shared/Recruit.Vacancies.Client/Domain/Entities/User.cs

```diff
@@ -1,15 +1,17 @@
-﻿using System;
-
-namespace Esfa.Recruit.Vacancies.Client.Domain.Entities
-{
-    public class User
-    {
-        public Guid Id { get; set; }
-        public string IdamsUserId { get; set; } 
-        public UserType UserType { get; set; }
-        public string Name { get; set; }
-        public string Email { get; set; }
-        public DateTime CreatedDate { get; set; }
-        public DateTime LastSignedInDate { get; set; }
-    }
-}
+﻿using System;
+using System.Collections.Generic;
+
+namespace Esfa.Recruit.Vacancies.Client.Domain.Entities
+{
+    public class User
+    {
+        public Guid Id { get; set; }
+        public string IdamsUserId { get; set; } 
+        public UserType UserType { get; set; }
+        public string Name { get; set; }
+        public string Email { get; set; }
+        public DateTime CreatedDate { get; set; }
+        public DateTime LastSignedInDate { get; set; }
+        public IList<string> AccountsDeclaredAsLevyPayers { get; set; } = new List<string>();
```

Comment:
We're only remembering accounts that have declared themselves as levy payers.  So if you are a non-levy payer then for each session you are always going to be asked the question about being a levy payer or not.  Is this right?

Reply:
Yep that's correct. Any request to a page will redirect you to the choice again.